app/src/main/java/com/example/fitappa/View/ViewRoutinesActivity.java

```diff
@@ -25,9 +26,10 @@ protected void onCreate(Bundle savedInstanceState) {
         // Load the elements
         routinesLayout = findViewById(R.id.routinesLayout);
         Button createRoutineBtn = findViewById(R.id.CreateRoutineBtn);
+        Button back = findViewById(R.id.backButton);
```

Comment:
would be nicer to make this backBtn (following the convention of other variables of type Button)